CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+## Contributing Guide
+
+### Getting Started
+
+This guide will help you setup and contribute to jovian.
+
+**Step 1: Clone the repo:**
+
+```
+git clone git clone https://github.com/jvn-io/jovian-py.git
+```
+
+`cd` into the `jovian-py` folder:
+
+```
+cd jovian-py
+```
+
+**Step 2: Setup the Python environment for devolopment:**
+
+Create a new Python environment using [conda](conda.io).
+
+_Make sure you have [conda](https://docs.conda.io/en/latest/) setup on your pc_
+
+```
+conda create -n jovian-py-dev python=3.5 -y
+```
+
+Activate the environment:
+
+```
+conda activate jovian-py-dev
+```
+
+Install the dependencies by running:
+
+```
+pip install -r requirements.txt
+```
+
+To ensure that VSCode and other code editors use the correct conda environment, run the following command to add the path of the `jovian-py-dev` env to your `.bashrc` file:
+
+```
+echo -e "\n\nexport JOVIAN_PY_DEV_PYTHONPATH=$CONDA_PREFIX/bin/python" >> ~/.bashrc
+
+source ~/.bashrc
+```
+
+The installation of dependencies is complete.
+When working on jovian, please ensure that you activate the `jovian-py-dev` environment inside the `jovian-py` folder. You can deactivate the environment using:
+
+```
+conda deactivate
+```
+
+**Step 3: Install CMake:**
+
+Make sure you have `CMake` [installed](https://cmake.org/install/)
+
+To test the installation, run the following command:
+
+```
+make help
+```
+
+This should print out:
+
+![](https://i.imgur.com/9yFX3oh.png)
+
+**Step 4: Setup Code Editor:**
+
+We recommend using [VSCode](https://code.visualstudio.com/),
+
+_Make sure VSCode can be [launched from the command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)_
+
+We recommend using the following extensions in VSCode:
```

Comment:
You can add link or explain how to do add these extensions from the VSCode UI itself.